docs/DeveloperGuide.md

```diff
@@ -1,15 +1,232 @@
 # Developer Guide
-
+This developer guide will help you to get started with our product, badMaths!
+
+## Table of Contents
+
+1. [Acknowledgements](#acknowledgements) 
+2. [Design & implementation](#design--implementation)
+   - [Store Notes](#store-notes) 
+   - [Graph](#graph)
+   - [Matrix](#matrix)
+2. [Product Scope](#product-scope)
+   - [Target user profile](#target-user-profile)
+   - [Value proposition](#value-proposition)
+3. [User Stories](#user-stories)
+4. [Non-Functional Requirements](#non-functional-requirements)
+5. [Glossary](#glossary)
+6. [Instructions for manual testing](#instructions-for-manual-testing)
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
+
 ## Design & implementation
+### Solving Quadratic Equations
+The Quadratic equation solver allows user to find the value of `x` in a simple quadratic equation.
+The function is started when `solveQuadratic()` is called.
+Below is an  example of how it would be called.
+```
+Quadratic. 2x^2 + 3x + 1
+```
+Step 1. Parser class would split the user input into 2 parts, command and toDo. Command would be `Quadratic`,
+thus `executeCommand` would trigger `quadraticSolver`. toDo would be `2x^2 + 3x + 1`.
+
+Step 2. The general formula of quadratic equations is `ax^2 + bx + c`, thus `findA`, `findB`, `findC`
+would identify a, b and c from toDo.
+
+Step 3. To solve the quadratic equation, a, b and c are put into the formula `x = (-b±√(b^2-4ac))/2a`.
+
+
+### Store and Display Notes
+The Store and Display Notes functionality allows users to input notes 
+which will be stored in a list, and to display a list of all notes
+stored by users. This functionality is achieved through the `Store.` and `List.` commands.
+
+The syntax for the commands of this functionality is as follows:
+1. `Store. <item>` For storing of notes into the list, where `<item>` represents the note that 
+users would like to store. 
+   Example: `Store. addition`
+2. `List.` For displaying of a list of all notes stored by users.
+   Example: `List.`
+
+Below is an illustration of how the `Store.` and `List.` are used to allow
+users to input notes to a list and to display a list of all notes that are
+stored inside the list.
+1. This is step 1.
+2. This is step 2.
+
+![img.png](img.png)
```

Comment:
Is this format of the sequence diagram correct? Should there be colon in the class such as :Notes?